### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by fira959**
Maybe the addition of @HOME/.mail should probably be move into an abstraction with the alternative common mail dir locations since they are shared by all mail clients

### CODE REVIEW

**Comment by roddhjav** [comment]
It already exist (see in `apparmor.d/tunables/home.d/apparmor.d`). You may want to edit `XDG_MAIL_DIR` to `.mail`. Then, you can use `@{user_mail_dirs}` in the profiles.

File: apparmor.d/profiles-m-r/mutt [hunk 1]
```diff
@@ -97,7 +98,7 @@ profile mutt @{exec_path} {
     @{bin}/lynx    mrix,
     
     owner @{HOME}/.w3m/* rw,
-
+    owner @{user_mail_dirs} rw,
```
Comment (roddhjav) [inline apparmor.d/profiles-m-r/mutt:101]: You need `owner @{user_mail_dirs}/{,**} rw,` (same elsewhere)

**Comment by doublez13** [comment]
For the Mutt profile, You're granting all the the child profiles `rw` access to `owner @{user_mail_dirs}`. Why?
They should not have access to this.

**Comment by fira959** [comment]
> For the mutt profile, You're granting all the the child profiles `rw` access to `owner @{user_mail_dirs}`. Why? They should not have access to this.

the html-renderer subprofile needs read access to files like ~/.mail/inbox and write access to ~/.mail/tmp/

But the other subprofiles actually didn't need any access like this, not sure why I included them. Permissions should be minimal now

**Comment by doublez13** [comment]
Thanks for updating that :)

Have you overridden a setting in you're mutt config or maybe a custom mailcap definition?
I'm still unclear why the `html-renderer` profile would need any access to your mail directory, as mutt and the html renderer programs would normally communicate via the file `/{var/,}tmp/{.,}mutt*`

If this is something nonstandard that you have overridden in your settings, the rules should live in your personal `local/mutt_html-renderer` file.

Or maybe I'm missing something obvious :)

**Comment by fira959** [comment]
> Thanks for updating that :)
> 
> Have you overridden a setting in you mutt config or maybe a custom mailcap definition? I'm still unclear why the `html-renderer` profile would need any access to your mail directory, as mutt and the html renderer programs would normally communicate via the file `/{var/,}tmp/{.,}mutt*`
> 
> If this is something nonstandard that you have overridden in your settings, the rules should live in your personal `local/mutt_html-renderer` file.
> 
> Or maybe I'm missing something obvious :)

You are correct that I am using a setting for the tmp file that points to that directory. It is not the default but if I recall it is a common recommendation to avoid leaking private data. It could therefore be argued to be a custom change even if it is common. I dont see the harm in allowing access to the tmp folder in any @{user_mail_dirs}. @roddhjav Your call

**Pull request merged**